packages/teams-js/src/private/messageChannels.ts

```diff
@@ -0,0 +1,59 @@
+import { requestPortFromParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { runtime } from '../public/runtime';
+
+/**
+ * @hidden
+ * Namespace to request message ports from the host application.
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export namespace messageChannels {
```

Comment:
This namespace and all of the exported functions should be marked with the `@beta` tag, which will let people know that the signatures here may change with little or no notice. Once you have this in production and are happy with it / feel confident you won't need to make breaking changes you can come back and remove the `@beta` tag.

Reply:
Done